### DISCUSSION TITLE
Make mouse back button close tab when unable to go back

### DISCUSSION THREAD

**Discussion by mr-cheffy** [zen-browser/desktop#835] [Ideas]
### What feature would you like to see?

Currently, when I open a new tab or open a link in a new tab, pressing the back button has no effect. It would be a helpful feature to allow users to easily close tabs without needing to move the cursor to the tab bar.

### What platform are you seeing the problem on?

Linux

 
 ~ _Created automatically from issue: https://github.com/zen-browser/desktop/issues/442_

**Comment by fulldecent** [comment]
Not just close tab. This should specifically go back to the tab which caused to the closing tab to be opened.

Or in other words, if you put a sticker on the left side of your screen (and didn't know what tabs are because sometimes navigating will be in the same tab or sometimes it will be in a new tab) then the back button would do the same thing whether you got to that page by new tab or not.

---

Supporting this feature will achieve parity with Arc.

**Reply by mbugbee** [reply]
This would an awesome add.  I was using an extension to add this functionality but it seems to no longer be working for me.

**Comment by LaserKaspar** [comment]
Very similar to this if not the same! This would be a great addition.
https://github.com/zen-browser/desktop/discussions/5911

**Comment by Wabuo** [comment]
I just found a Firefox add-on that kinda provides this service.

It's not perfect, but for me, it's good enough.


https://addons.mozilla.org/en-US/firefox/addon/back-to-close-we/
